home/tests/test_api.py

```diff
@@ -462,93 +464,95 @@ def test_detail_view_no_content_page(self, uclient):
 @pytest.mark.django_db
 class TestWhatsAppMessages:
     """
-    FIXME:
-     * Should some of the WhatsApp tests from TestPagination live here instead?
```

Comment:
I kept these separate because they test the WhatsApp specific functionality where the WhatsApp tests in TestPagination tests the more generic things but just using the WhatsApp channel. 

Let me know if you think otherwise.

Reply:
Yeah, I think that was the intention and I just expressed it poorly in the FIXME.